tensorboard/pip_package/requirements.txt

```diff
@@ -26,7 +26,7 @@ numpy >= 1.12.0
 # At the same time, any constraints we specify here must allow at least some
 # version to be installed that is also compatible with TensorFlow's constraints:
 # https://github.com/tensorflow/tensorflow/blob/9d22f4a0a9499c8e10a4312503e63e0da35ccd94/tensorflow/tools/pip_package/setup.py#L100-L107
-protobuf >= 3.19.6
+protobuf >= 3.19.6, < 4.24
```

Comment:
Perhaps leave a comment with a reference to a bug?

Reply:
Done. Let me know if you have suggestions for a different phrasing.

Reply:
Looks good.